distributions/build-debian-package.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+sudo apt-get install devscripts build-essential lintian dh-make
+sudo apt-get install libqt4-dev libjack-jackd2-dev
```

Comment:
Hi Marc, the compile info on the Jamulus homepage is a bit outdated. Jamulus should be compiled with Qt5 instead of Qt4. I am not 100 % sure what is the aquivalent devel package for Qt5 as it was libqt4-dev for Qt4. Do you know the name of the packet?

Reply:
ok, i will checkt that, its maybe qtdeclarative5-dev, but i must test first, i took the installed packages from https://github.com/corrados/jamulus/blob/master/INSTALL#L42 so you shold change that too. after i tested successfully i will answer here what exact library is needed and if it works with qt5

Reply:
If you like you can do the change in the INSTALL file yourself. The only thing I have to do afterwards is to update the Jamulus Homepage as well.

Reply:
well I am still testing, i have some problems with qt5/qjackctl in virtualBOX, still debugging it. Because otherwise I would need to re-setup computers for each test. So I prefere to do the tests on a virtualBOX image, so I can use snapshots. Still not working. i will tell you the results... btw: I like challenges -- at least those sort of challenges

Reply:
didn't work with virtualBOX, so tested the with new physical installation.